buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/PerformanceTest.java

```diff
@@ -85,12 +88,15 @@ public void setScenarios(String scenarios) {
         this.scenarios = scenarios;
     }
 
-    @Nullable @Optional @Input
+    @Nullable
```

Comment:
Now we've code style checked in, so let's reformat them with our unified code style configuration.